src/db/index.js

```diff
@@ -1,5 +1,47 @@
-export default async function makeDb() {
+import axios from 'axios'
+
+const DATAPOINT_URL = 'https://mock-dbapi.herokuapp.com/api'
```

Comment:
For this, 
1. rename `DATAPOINT_URL` to `BASE_URL`
2. In /src/config/environment/index.js file, add 
```javascript
BASE_URL: process.env.BASE_URL
```
3. In the object being exported, create a function called `getEnvironment()`. 
4. In the `getEnvironment()` function, determine the current NODE_ENV and return the right BASE_URL
5. In the `src/db/index.js` file, import the `getEnvironment()` function, depending on the environment we are in, the base_url either points to zc_core API or our test environment,
>BASE_URL either points to the zc_core API or our test db.

Reply:
There's no api documentation for the Zc_core, so I am not sure if it'll respond in the same format as the mock db api. I'll refactor once a comprehensive api documentation is out.